lib/gatling/init_script_template.sh.eex

```diff
@@ -13,6 +13,6 @@ export MIX_ENV=prod
 export PORT=<%= @port %>
 export HOME=<%= System.user_home %>/deployments/<%= @project_name %>
 
-export $(cat <%= System.user_home %>/deployments/<%= @project_name %>/.env)
+[ -f $HOME/.env ] && export $(cat $HOME/.env)
```

Comment:
Sure, I haven't added it, I just used the one that was already there. :)

Dunno where/how it's used by the app.